docs/gatsby-node.js

```diff
@@ -1,9 +1,5 @@
-const defines = require('../babel-defines')
-
 exports.onCreateWebpackConfig = ({actions, plugins, loaders, getConfig}) => {
   const config = getConfig()
-  // Add our `__DEV__` and `process.env.NODE_ENV` defines
-  config.plugins.push(plugins.define(defines[process.env.NODE_ENV || 'development']))
```

Comment:
Check out the note above: https://github.com/primer/components/pull/1045/files#r574817860